python/cugraph/cugraph/tests/test_node2vec.py

```diff
@@ -172,14 +96,183 @@ def test_node2vec_invalid(
     for bad_depth in invalid_max_depths:
         with pytest.raises(ValueError):
             df, seeds = calc_node2vec(G, start_vertices, max_depth=bad_depth,
-                                      use_padding=use_padding, p=p, q=q)
+                                      compress_result=compress, p=p, q=q)
     # Tests for invalid p
     for bad_p in invalid_pqs:
         with pytest.raises(ValueError):
             df, seeds = calc_node2vec(G, start_vertices, max_depth=max_depth,
-                                      use_padding=use_padding, p=bad_p, q=q)
+                                      compress_result=compress, p=bad_p, q=q)
     # Tests for invalid q
     for bad_q in invalid_pqs:
         with pytest.raises(ValueError):
             df, seeds = calc_node2vec(G, start_vertices, max_depth=max_depth,
-                                      use_padding=use_padding, p=p, q=bad_q)
+                                      compress_result=compress, p=p, q=bad_q)
+
+
+# Remove once bug is resolved
```

Comment:
At that moment, I was under the impression that the toy line graph would not remain once the PR left draft status, but since its utility can be used in other algos, it can go either way. I removed the comment for this next commit, but I can remove the test if the line graph example is also to be gone.